README.md

```diff
@@ -1,5 +1,20 @@
 # Pyocient Release Notes
 
+## 3.0.2
+
+- The `TypeCodes` class has been changed to a python `IntEnum`
+- GIS types (STPoint, STLinestring, STPolygon) have been renamed from _STPoint, _STLinestring, _STPolygin
```

Comment:
@cj81499 is this a breaking change to move a "private" type to a public name? I don't think so?